deis-dev/tpl/deis-objectstorage-secret.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: objectstorage-keyfile
+  namespace: deis
+  labels:
+    heritage: deis
+  annotations:
+    deis.com/objectstorage: "{{.storage}}"
```

Comment:
Elsewhere, we have used deis.io (or subdomains thereof) as a prefix.  I vaguely recall some short discussion about this a couple months back where we had agreed .io was better in this context because it was more open-sourcey.  That aside, now that .io's been used elsewhere, it's more a matter of consistency now.